ddsreplayer/src/cpp/tool/DdsReplayer.cpp

```diff
@@ -122,28 +113,24 @@ DdsReplayer::DdsReplayer(
             configuration.replayer_configuration->app_metadata,
             "true");
 
-        // Set as server in TypeLookup service
-        pqos.wire_protocol().builtin.typelookup_config.use_client = false;
-        pqos.wire_protocol().builtin.typelookup_config.use_server = true;
-
         // Participant creation via factory
```

Comment:
Comment for the reviewer: The `DynTypesParticipant` has been removed from the Replayer. Its sole purpose was to manage the `DynamicTypes`, but now this responsibility has been transferred to the participants within the `DDSPipe`.